src/boot/bootctl-install.c

```diff
@@ -241,7 +241,7 @@ static int version_check(int fd_from, const char *from, int fd_to, const char *t
         return 0;
 }
 
-static int copy_file_with_version_check(const char *from, const char *to, bool force) {
+static int copy_file_with_version_check(const char *from, const char *to, bool force, bool ignore_no_version) {
```

Comment:
our usual rule is that once a function gets more than a single "bool" parameter that is a flag you have to convert it to a proper flags param, i.e. change this to:

```c
typedef enum CopyWithVersionFlags {
        COPY_WITH_VERSION_FORCE                  = 1 << 0,
        COPY_WITH_VERSION_IGNORE_MISSING_VERSION = 1 << 1, 
} CopyWithVersionFlags;

static int copy_file_with_version_check(const char *from, const char *to, CopyVersionFlags flags) {
```

you get the idea